### ISSUE TITLE
getNewAuthCode results in error

### ISSUE THREAD

**Issue by michaeldewal** [dburles/meteor-two-factor#3]
The following event results in an error. The normal code generation works perfectly

```
'click #resend-security-code'(e) {
      e.preventDefault();
      twoFactor.getNewAuthCode(error => {
        if (error) {
          bootbox.alert(error.reason);
          this.state.securityBtnCheckWaiting = false;

        }
      });
    }
```

`Exception in delivering result of invoking 'twoFactor.getAuthenticationCode': TypeError: handlerCb is not a function`

**Comment by dburles** [comment]
Hey @michaeldewal thanks, you're right, I'll have a fix up shortly

**Comment by dburles** [comment]
Fixed in 1.1.1
